tests/modules/metabat2/metabat2/test.yml

```diff
@@ -0,0 +1,19 @@
+- name: metabat2 metabat2 test_metabat2_no_depth
+  command: nextflow run tests/modules/metabat2/metabat2 -entry test_metabat2_no_depth -c tests/config/nextflow.config
+  tags:
+    - metabat2
+    - metabat2/metabat2
+  files:
+    - path: output/metabat2/bins/test.1.fa.gz
+    - path: output/metabat2/test.depth.txt.gz
+    - path: output/metabat2/test.membership.tsv.gz
```

Comment:
```suggestion
    - path: output/metabat2/test.tsv.gz
```

Reply:
If you want to add `membership`, you can update the `options` definitions in the `include` statment of the tests to include the prefix